gravitl/netclient
# Gra 985/host updates logic

**Review after PR #133** is merged
-> Host update logic

Testing Steps:- @theguy951357 
Update host settings on UI and verify if it's getting reflected on host.
--> for example: change listen port on UI and and check whether interface is reset to newport
